libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/EmailProvider.java

```diff
@@ -16,19 +16,18 @@
 
 package com.yahoo.athenz.common.server.notification;
 
+import javax.mail.internet.MimeMessage;
 import java.util.Collection;
 
 public interface EmailProvider {
 
     /**
      * Send an email through the provider
-     * @param subject
-     * @param body
      * @param status
      * @param recipients
      * @param from
-     * @param logoImage
+     * @param mimeMessage
      * @return true if mail was sent
      */
-    boolean sendEmail(String subject, String body, boolean status, Collection<String> recipients, String from, byte[] logoImage);
```

Comment:
I have changed the interface but it is unlikely that someone already implemented a new EmailProvider.